sale_order_import/templates/error_lines_chatter_msg.xml

```diff
@@ -0,0 +1,42 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<odoo>
+    <template id="template_skip_error_lines_message">
+        <h4 class="text-danger">Errors lines on Import: <t
+                t-out="len(lines)"
+            /> line(s)</h4>
+        <t t-foreach="lines" t-as="line" t-key="line_index">
```

Comment:
What is `t-key`? Inside a loop you always have `line_index`.
https://www.odoo.com/documentation/17.0/developer/reference/frontend/qweb.html#loops